src/aiocomelit/api.py

```diff
@@ -87,6 +87,9 @@ class ComelitVedoZoneObject:
 class ComelitCommonApi:
     """Common API calls for Comelit SimpleHome devices."""
 
+    _json_data: ClassVar[list[dict[Any, Any]]] = [{}, {}, {}, {}, {}]
```

Comment:
Why is this object shared between all instances?

Reply:
Previous class design, updated.